hr_holidays_natural_period/__manifest__.py

```diff
@@ -12,5 +12,5 @@
     "installable": True,
     "depends": ["hr_holidays"],
     "maintainers": ["victoralmau"],
-    "demo": ["demo/hr_leave_type_data.xml"],
+    "demo": ["demo/hr_leave_type_demo.xml"],
```

Comment:
This change requires a migration script to try to delete the type.

Reply:
I have left the file name as it was.

Reply:
No, it's correct to be demo data, as it was an error to put it as data, but it should delete old occurrences through delete_safely... method in openupgradelib.

Reply:
I'm sorry but I don't understand.

They have always been defined as demo data, now we are just renaming the file to make it more consistent.

Reply:
Ah, OK, my fault. Sorry, I saw data > demo, and I think this is changing the other part (the key, changing its meaning). It's OK then to just rename and nothing more.